consensus/obcpbft/pbft-core.go

```diff
@@ -668,6 +671,17 @@ outer:
 		}
 		logger.Debug("Replica %d has detected request %s must be resubmitted", instance.id, d)
```

Comment:
As there is currently no log level checking at all in the obcpbft package, I've opened #1770 to track this.